content/post/writing-a-book-remotely.md

```diff
@@ -0,0 +1,576 @@
+---
+title: "Writing a Book Remotely Without Ever Meeting In Person Before"
+date: 2022-04-12T09:27:17+02:00
+description: "TODO"
+keywords: []
+cover:
+draft: false
+tags: []
+math: false
+toc: false
+---
+
+Ideas for the main title:
+(looking for a more positive word for lessons learned, Retrospective?)
+- Lessons Learned: Writing a Technical eBook Remotely Without Ever Meeting In Person Before
+- Lessons Learned: Writing a Technical eBook Remotely
+- Interconnected World: Writing a Technical eBook Remotely
+- How To Write a Technical eBook Remotely Without Every Meeting
+- How we managed to write a book about AWS remotely
+
+This article describes a one-year journey of writing [Stratospheric - From Zero to Production with Spring Boot and AWS](https://stratospheric.dev/) as a co-author with two other co-authors that I met on Twitter. We never met in person before and managed to successfully write a 450+ in-depth guide on developing and deploying Java Spring Boot applications on AWS (Amazon Web Services).
+
+In short, this article covers: ,
+- How we self-published this eBook while writing the manuscript in Markdown
+- How we promoted and sold the eBook
+- How we organized ourselves to write the eBook in an asynchronous fashion while having 8hour time span
+- General tips, recommendations and lessons learned
+
+The whole project made (and still makes) a lot of fun. I learned a ton about AWS and got the chance to connect and work with knowledgeable and nice people.
+
+The three co-authors are: [Tom](https://twitter.com/TomHombergs/) (🇩🇪 living in 🇦🇺), [Björn](https://twitter.com/bwilmsmann/) (🇩🇪), [Philip](https://twitter.com/rieckpil) (🇩🇪 - that's me).
+
+Let's get started 🚀
+
+## How We Got Together 👥
+
+> Luck is what happens when preparation meets opportunity.
+
+While scrolling through my Twitter feed I saw a fellow technical blogger reaching out to other bloggers to organize a small knowledge exchange.
+
+Small anecdote how it started with a simple Tweet by Tom.
+
+{{< tweet user="TomHombergs" id="1278635778722000902" >}}
+
+We next met with the intent to share experience as tech bloggers. During the second or third meeting of this kind, Tom asked for fellow bloggers that would be interested in taking a closer look at AWS and Spring Boot and may want to write about it.
+
+Björn and me who were participating in this meeting raised our hands. We've never met each other before. And partially knew each other from reading our blogs.
+
+We organized a separate meeting to discuss more details and to get to know each other.
+
+## Our Goals For This Project ⭐️
+
+Before we started to work on this project we met several times to get to know each other and align our goals and expectations. While we were all three working full-time for clients/an employer, we set the common understanding that this will be a fun side project.
```

Comment:
```suggestion
Before we started to work on this project we met several times to get to know each other and align our goals and expectations. While all three of us were working full-time for clients/an employer, we set the common understanding that this will be a fun side project.
```